gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/vertx/web/handler/impl/internal/mfa/MfaFilterContext.java

```diff
@@ -125,7 +125,7 @@ public boolean isDeviceRiskAssessmentEnabled() {
     }
 
     public boolean isValidSession() {
```

Comment:
**TODO**: if we are testing the CHALLENGE_COMPLETED_KEY instead of a AUTH_COMPLETED, then this method should be renamed IMO.

Reply:
ok, renamed